UnitySDK/Assets/ML-Agents/Editor/Builder.cs

```diff
@@ -0,0 +1,15 @@
+﻿#if UNITY_CLOUD_BUILD
+
+namespace MLAgents
+{
+
+	public class Builder
+	{
+		public static void PreExport()
+		{
+			BuilderUtils.SwitchAllLearningBrainToControlMode();
```

Comment:
I think we should set the brains to control when we launch the executable, not when we build it.